Extensions/find_inactives.js

```diff
@@ -165,19 +173,42 @@ XKit.extensions.find_inactives = new Object({
 		var time_value = updated_time.replace(/[^0-9\.]/g, '');
 		time_value = parseInt(time_value, 10);
 
-		if(updated_time.indexOf("day") > -1){
-			return time_value;
+		for(var i = 0; i < this.language_date_strip.length; i++) {
+			updated_time = updated_time.replace(this.language_date_strip[i], '');
 		}
-		if(updated_time.indexOf("week") > -1){
-			return time_value * 7;
+
+
+		var days = this.calculate_days(updated_time, time_value, this.language_date_day, 1);
```

Comment:
This `days_since_updated` function has a lot of duplicate code that can be refactored.